ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -10,7 +10,7 @@
    [cmr.common.time-keeper :as time-keeper]
    [cmr.common.util :as util]
    [cmr.common.validations.json-schema :as js]
-   [cmr.ingest.config :as ingest-config]
+   [cmr.common-app.config :as common-app-config]
```

Comment:
nit: We sort required namespaces in alphabetic order. Move this line up. This comment applies to other files too.